### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by lizalc**

### CODE REVIEW

**Comment by lizalc** [comment]
Here's the MIT license. I'll merge it to master once it's been approved. Basically one of you just needs to look at the "Files changed" then click the "Review Changes" button. This workflow is the same for the other pull requests, so those should be reviewed and approved before the demo day that way everything is merged into `master` by then.

**Review by Janak-Patel — APPROVED** [review]

**Pull request merged**
